docs/iris/src/whatsnew/contributions_2.0/incompatiblechange_2017-Oct-24_netcdf-no-unlim.txt

```diff
@@ -0,0 +1,8 @@
+* Deprecated FUTURE flag :attr:`iris.Future.netcdf_no_unlimited`.
+
+* Removed deprecated behaviour that automatically set the length of the outer netCDF variable to 'UNLIMITED' on save.
+  This change means that no cube dimension coordinates will be automatically saved as netCDF variables with 'UNLIMITED' length.
+  You can manually specify cube dimension coordinates to save with 'UNLIMITED' length.
```

Comment:
I find using the second-person a bit jarring in a technical context. Can you change this to the passive voice? Something like 'Cube dimension coordinates to save with 'UNLIMITED' length can be manually specified.'

Reply:
Or 'It's possible to specify...'